op-e2e/system_adminrpc_test.go

```diff
@@ -172,6 +177,54 @@ func TestLoadSequencerStateOnStarted_Started(t *testing.T) {
 	assertPersistedSequencerState(t, stateFile, node.StateStarted)
 }
 
+func TestPostUnsafePayload(t *testing.T) {
+	InitParallel(t)
+	ctx := context.Background()
+
+	cfg := DefaultSystemConfig(t)
+	cfg.Nodes["verifier"].RPC.EnableAdmin = true
+
+	sys, err := cfg.Start(t)
+	require.NoError(t, err)
+	err = sys.BatchSubmitter.Stop(ctx)
+	require.NoError(t, err)
+	defer sys.Close()
+
+	l2Seq := sys.Clients["sequencer"]
+	l2Ver := sys.Clients["verifier"]
+	rollupRPCClient, err := rpc.DialContext(ctx, sys.RollupNodes["verifier"].HTTPEndpoint())
+	require.Nil(t, err)
+	rollupClient := sources.NewRollupClient(client.NewBaseRPCClient(rollupRPCClient))
```

Comment:
I know you're following what's already in this file, but you can now use `sys.RollupClient("verifier")` to get a rollup client and the test will manage its lifecycle properly.
```suggestion
	rollupClient := sys.RollupClient("verifier")
```